core/services/ocr2/plugins/mercury/integration_test.go

```diff
@@ -539,6 +540,14 @@ func integration_MercuryV2(t *testing.T) {
 	bootstrapNode := Node{App: appBootstrap, KeyBundle: bootstrapKb}
 	logObservers = append(logObservers, observedLogs)
 
+	// Bury it with finality depth
```

Comment:
maybe some explanation would be helpful

Reply:
Replacing with, "Commit blocks to finality depth to ensure LogPoller has finalized blocks to read from."